code/datums/gamemode/vampire_gamemode.dm

```diff
@@ -1,38 +1,77 @@
 /datum/gamemode/vampire
-	minimum_player_count = 10
+	minimum_player_count = 0 // OVERRIDE
 	name = "Vampire"
 	factions_allowed = list(/datum/faction/vampire)
 
+	roles_allowed = 1 // Per factions
+
+	var/number_of_vamps
```

Comment:
Surely there's a better way than having explicit variables per gamemode for each antag type.